addons/website/static/src/scss/bootstrap_overridden.scss

```diff
@@ -83,6 +83,14 @@ $link-hover-color: auto-contrast(darken($link-color, 15%), $body-bg, 'o-cc1-link
 $link-decoration: if(o-website-value('link-underline') == 'always', underline, none) !default;
 $link-hover-decoration: if(o-website-value('link-underline') != 'never', underline, none) !default;
 
+// Emphasis
+//
+// For higher contrast text. Not applicable for backgrounds.
+
+// Increase contrast when needed, otherwise fallback to BS default
+$primary-text-emphasis: if(not has-enough-contrast(o-color('o-color-1'), $body-bg), increase-contrast(o-color('o-color-1'), $body-bg), null) !default;
+$secondary-text-emphasis: if(not has-enough-contrast(o-color('o-color-2'), $body-bg), increase-contrast(o-color('o-color-2'), $body-bg), null) !default;
```

Comment:
If I am not mistaken, you can (and thus should) use `o-color('primary')` and `o-color('secondary')` here, not relying on the coupling that is done elsewhere